metadata.json

```diff
@@ -11,6 +11,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 8.0.0 < 9.0.0"
+    },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 9.0.0 < 10.0.0"
```

Comment:
concat 9.x requires stdlib 9.x. Either allow older concat versions as well, or allow a newer stdlib

Reply:
ty, i've added that